velox/vector/tests/CMakeLists.txt

```diff
@@ -11,81 +11,85 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-add_library(velox_vector_test_lib VectorMaker.cpp VectorTestBase.cpp)
+if(${VELOX_BUILD_TEST_UTILS})
+  add_library(velox_vector_test_lib VectorMaker.cpp VectorTestBase.cpp)
```

Comment:
These libraries are required even with VELOX_BUILD_TESTING ; I suspect This is why xsimd doesnt get picked up as its header only.